packages/react-components/src/components/VideoTile.test.tsx

```diff
@@ -28,4 +35,22 @@ describe('VideoTile', () => {
     /* @conditional-compile-remove(pinned-participants) */
     expect(mockCallback).toBeCalledTimes(1);
   });
+
+  test('VideoTile shows more button when contextualMenu is set', async () => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const videoTileProps = { displayName: 'John Doe', contextualMenu: { items: [] } } as any;
```

Comment:
Just curious why we show an icon button if an empty array of items is provided? What does it look like when you click the menu button?

Reply:
Since we are now allowing Contoso devs to inject the ContextualMenu, we can't enforce a check using an internal object inside that type. Hence, it is unto Contoso to ensure that `contextualMenu` is undefined if they don't want the icon to show.